hyperspy/_signals/eds_tem.py

```diff
@@ -287,30 +289,53 @@ def quantification(self,
                        intensities,
                        method,
                        factors,
-                       composition_units='atomic',
-                       navigation_mask=1.0,
-                       closing=True,
-                       plot_result=False,
+                       composition_units = 'atomic',
+                       absorption_correction = False,
+                       tilt_stage = 0,
+                       take_off_angle = 'auto',
+                       thickness = None,
+                       conv_crit = 0.5,
+                       navigation_mask = 1.0,
+                       closing = True,
+                       plot_result = False,
+                       probe_area = 'auto',
+                       max_iterations = 30,
```

Comment:
I have thickness=auto do we need mass_thickness as well?

Reply:
No. I wrongly suggested this at the time of the review because there was a `mass_thickness` parameter in the docstring (which was not correct), but it has fixed by correcting the docstring.